cypress/integration/filter-group-examples/index.spec.ts

```diff
@@ -94,11 +94,6 @@ describe('Custom date/time pattern example', () => {
     cy.get('@filterGroupWrapper').find('tr').should('have.length', 3);
   });
 
-  it('opens the popover when click the button', () => {
```

Comment:
This keeps flaking on everyone's PR and is tested elsewhere. Removing to solve the flake.